modules/post/multi/gather/tomcat_gather.rb

```diff
@@ -150,16 +181,21 @@ def run()
       gathernix()
     end
 
+  
+  if $username.size == 0
+    print_status("No user credentials have been found")
+  end
+
   i=0
   while i < $username.count
     print_good("Username and password found: " + $username[i] + ":" + $password[i])
```

Comment:
Either this line should include the file path, or a header should be printed when you loop through the files so that we know which file the un/pass came from